ban-options.php

```diff
@@ -269,7 +269,7 @@ function toggle_checkbox() {
         </tr>
         <tr>
             <td><?php _e('User Agent', 'wp-ban'); ?>:</td>
-            <td><strong><?php echo $_SERVER['HTTP_USER_AGENT']; ?></strong></td>
+            <td><strong><?php echo (!isset($_SERVER["HTTP_USER_AGENT"]) ? __('Unknown', 'wp-ban') : htmlentities(strip_tags($_SERVER['HTTP_USER_AGENT']))); ?></strong></td>
```

Comment:
I think we can use esc_attr, make use of WP functions

`<?php echo ! isset($_SERVER['HTTP_USER_AGENT'] ? __( 'Unknown', 'wp-ban' ) : esc_attr( $_SERVER['HTTP_USER_AGENT'] ); ?>`

Reply:
Definitely the best choice, would use **esc_html** instead

Will update the PR in a few minutes